modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/OrganizationStagedModelDataHandler.java

```diff
@@ -345,6 +365,33 @@ private void _importAddresses(
 			importedOrganization.getOrganizationId(), addresses);
 	}
 
+	private Organization _importCountry(
+			PortletDataContext portletDataContext, Organization organization)
+		throws Exception {
+
+		List<Element> countryElements =
+			portletDataContext.getReferenceDataElements(
+				organization, Country.class);
+
+		if (!countryElements.isEmpty()) {
+			Element countryElement = countryElements.get(0);
+
+			String countryPath = countryElement.attributeValue("path");
+
+			Country country = (Country)portletDataContext.getZipEntryAsObject(
+				countryPath);
+
+			if (country != null) {
+				organization.setCountryId(
+					_countryLocalService.getCountryByA2(
+						portletDataContext.getCompanyId(), country.getA2()
+					).getCountryId());
```

Comment:
Here you are chaining and I don't think it's accepted. You need to get the country in a variable and then retrieve the id from it